Makefile

```diff
@@ -1205,9 +1311,10 @@ help: Makefile make/tools.mk
 
 ## targets           : print a list of all valid target platforms (for consumption by scripts)
 targets:
-	$(V0) @echo "Valid targets: $(VALID_TARGETS)"
-	$(V0) @echo "Target:        $(TARGET)"
-	$(V0) @echo "Base target:   $(BASE_TARGET)"
+	$(V0) @echo "Valid targets:    $(VALID_TARGETS)"
+	$(V0) @echo "Target:           $(TARGET)"
+	$(V0) @echo "Base target:      $(BASE_TARGET)"
+	$(V0) @echo "Combined targets: targets-group-1, targets-group-2, targets-group-3, targets-group-4, targets-group-test-and-rest"
```

Comment:
Why list the internal list names? The content of the lists are more interesting.